spec/octokit/client/hooks_spec.rb

```diff
@@ -131,4 +131,12 @@
       end
     end # .remove_org_hook
   end # with org hook
+
+  describe ".parse_payload", :vcr do
+    it "parses payload string" do
+      result = @client.parse_payload '{"action": "added", "scope": "team", "sender": {"site_admin": false}}'
```

Comment:
Let's drop one of those fixtures in `spec/fixtures` and load it with the `fixture` helper just for consistency.